pkg/minikube/minikube.go

```diff
@@ -32,8 +32,9 @@ var kubernetesVersion = "1.22.2"
 var minikubeVersion = 1.23
```

Comment:
If we use `knative` for the default cluster name, we should update L30 above too (actually, it might be good to just declare the variable in L30 and wait to assign its value in L37 below)

Reply:
Done with all the comments and added a little refactor to the flag function